internal/execute/testsys_test.go

```diff
@@ -152,6 +152,18 @@ func (s *testSys) Writer() io.Writer {
 	return s.currentWrite
 }
 
+func (s *testSys) WriteOutputIsTTY() bool {
+	return true
+}
+
+func (s *testSys) GetWidthOfTerminal() int {
+	return 0
+}
+
+func (s *testSys) GetEnvironmentVariable(name string) string {
+	return ""
```

Comment:
Implement as a map thats taken on tscInput and passed down

Reply:
Sure, I can do that, it just wasn't used by anything in this PR.